lib/src/webos_tokens/color_semantic.dart

```diff
@@ -0,0 +1,80 @@
+/*
+ * SPDX-FileCopyrightText: Copyright 2025 LG Electronics Inc.
+ * SPDX-License-Identifier: LicenseRef-LGE-Proprietary
+ */
+
+import 'color_semantic/background/background.dart';
+import 'color_semantic/on_background/on_background.dart';
+import 'color_semantic/on_surface/on_surface.dart';
+import 'color_semantic/scrim/scrim.dart';
+import 'color_semantic/stroke/stroke.dart';
+import 'color_semantic/surface/surface.dart';
+
+export 'color_semantic/background/background.dart';
+export 'color_semantic/on_background/on_background.dart';
+export 'color_semantic/on_surface/on_surface.dart';
+export 'color_semantic/scrim/scrim.dart';
+export 'color_semantic/stroke/stroke.dart';
+export 'color_semantic/surface/surface.dart';
+
+abstract class ColorSemantic {
+  const ColorSemantic();
+
+  Background get background;
+  OnBackground get onBackground;
+  Surface get surface;
+  OnSurface get onSurface;
+  Stroke get stroke;
+  Scrim get scrim;
+}
+
+class LightColorSemantic extends ColorSemantic {
```

Comment:
We are planning to extend themes not only light and dark. So why don't we have separate folders or files for each theme?